Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,50 @@
 import React from 'react';
 import './App.css';
 
+import NumberButton from './components/ButtonComponents/NumberButton';
+import ActionButton from './components/ButtonComponents/NumberButton';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
+
 const App = () => {
+
+  const displayCharacter = "0";
+
+  const zeroButton = "0";
+
+  const nums = [7, 8, 9, 4, 5, 6, 1, 2, 3];
+
+  const mathButtons = ["÷", "x", "-", "+", "="];
+
+  const clearButton = "clear";
+
+  const numClass = "num-button";
+  const actionClass = "action-button";
+  const displayClass = "calc-display";
+  const mathBtnClass = "math-button";
+
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className="app">
+      <div className="calculator-container">
+        <CalculatorDisplay text={displayCharacter} displayStyle={displayClass} />
+        <div className="calculator-buttons">
+          <div className="number-buttons">
+            <ActionButton text={clearButton} buttonStyle={actionClass} />
```

Comment:
Hopefully this is all making sense now. You'll get more experience with the project today.